### PR TITLE
Correct arguments description in mcsas3_cli_runner.py

### PR DESCRIPTION
**Pull request by MarcBHahn**
Correct description in mcsas3_cli_runner.py for the configuration files.

### CODE REVIEW

File: mcsas3_cli_runner.py [hunk 1]
```diff
@@ -113,15 +114,15 @@ def isWindows():
...
+        help="Path to the config file how to read the input data",
         # required=True,
     )
     parser.add_argument(
         "-r",
         "--resultFile",
         type=lambda p: Path(p).absolute(),
         default=Path(__file__).absolute().parent / "test.nxs",
-        help="Path to the file to create and store the McSAS3 result in",
+        help="Path to the config file how to create and store the McSAS3 result in",
```
Comment (toqduj) [inline mcsas3_cli_runner.py:125]: This is not quite right, it is the file where the initial data, the intermediate data, the metadata and the end result is stored in.

File: mcsas3_cli_runner.py [hunk 2]
```diff
@@ -131,7 +132,7 @@ def isWindows():
         default=Path(__file__).absolute().parent
         / "example_configurations"
         / "run_config_spheres_auto.yaml",
-        help="Path to the filename with the SAXS data",
+        help="Path to the config how with the MCS/fitting parameters",
```
Comment (toqduj) [inline mcsas3_cli_runner.py:135]: Would rename to "Path to the configuration file containing the model parameters"

**Review by toqduj — CHANGES_REQUESTED** [review]
Thanks! would just suggest a few small changes in the wording. (would've done it myself if I knew how in this interface, but I'll pass the buck back to you..)

**Comment by MarcBHahn** [comment]
Here is the updated version

**Pull request merged**